### ISSUE TITLE
Can't reach the paper performance when using tf version

### ISSUE THREAD

**Issue by andrewliao11** [openai/rllab#119]
Hi I got some problem in training ```hopper``` in Mujoco. According to the paper, I can achieve 1183 using TRPO, but I can only achieve around 500 by using the following code (I modified from [trpo_swimmer.py](https://github.com/openai/rllab/blob/master/examples/trpo_swimmer.py)):
```python
env = TfEnv(normalize(HopperEnv()))

policy = GaussianMLPPolicy(
    name='GP',
    env_spec=env.spec,
    # The neural network policy should have two hidden layers, each with 32 hidden units.
    hidden_sizes=(100, 50, 25)
)
baseline = LinearFeatureBaseline(env_spec=env.spec)
algo = TRPO(
    env=env,
    policy=policy,
    baseline=baseline,
    batch_size=4000,
    max_path_length=500,
    n_itr=args.num_iter,
    discount=0.99,
    step_size=0.01,
    snap_shot_dir=args.snap_shot_dir,
    snapshot_mode='gap',
    snapshot_gap=args.snapshot_gap,
)
algo.train()
```

I've also tried to use ```GaussianStrategy``` to encourage exploration. but still not works....

**Comment by dementrock** [comment]
1. Use a larger batch size. In the paper we used 50000.
2. Try multiple random seeds. The performance of RL experiments vary a lot across different runs.
3. What is the version of tensorflow you are using?
4. Have you tried running the theano code and see if that works?

**Comment by andrewliao11** [comment]
hi @dementrock , thank for the quick reply! 
I'm right now trying to increase my batch size up to 50000 and try different seed.
Since there's only trpo for cartpole in tf, I'm trying to use tf to reimplement in Mujoco environments.
btw my tf is 1.0.1.

**Comment by andrewliao11** [comment]
@dementrock Thank so much for the advices! It works. (I only increase the bath size
